### DISCUSSION TITLE
Docker unoplatform/wasm-build image

### DISCUSSION THREAD

**Discussion by MasterThreadInf** [unoplatform/uno#11157] [Q&A]
Is there any update available or will be availble for https://hub.docker.com/r/unoplatform/wasm-build? 
For .net 6.0, unoplatform/wasm-build doesn't  support and uno works from .net 6.0 onwards ,

**Comment by jeromelaban** [comment]
There isn't but you can now use the [official .NET images](https://learn.microsoft.com/en-us/dotnet/architecture/microservices/net-core-net-framework-containers/official-net-docker-images).
